mon-pix/app/components/qrocm-dep-solution-panel.hbs

```diff
@@ -15,15 +15,15 @@
             class="correction-qrocm__answer correction-qrocm__answer--paragraph {{block.inputClass}}"
             rows="5"
             value="{{block.answer}}"
-            id="{{block.input}}"
```

Comment:
Le package `ember-template-eslint` génère une erreur de lint si un textarea ou un input a, à la fois, un `id` et un `aria-label` 🤷🏻‍♂️

Ils en parlent ici : https://github.com/ember-template-lint/ember-template-lint/issues/1835
Soi-disant résolu, mais toujours le problème de mon côté.

Donc au final, le plus simple est de virer l'id, qui est vraiment inutile de toute façon.

Reply:
L'`id` servait à le relié à son label avec le `for`. :) Ici on ne le relie plus à son label donc on ne sait plus à quel champ correspond le textarea. L'aria-label ajoute un nouvelle info ("question passée"...) mais ne reprend pas le texte du label ! Je vois deux solutions : 
- Ajouter un aria-label qui reprend le texte du label et le complète avec `block.input`
- Rajouter l'id pour le lié à l'input et ajouter un `<span class="screen-reader-only>{{block.ariaLabel}}</span>` dans le label

Reply:
![image](https://github.com/1024pix/pix/assets/7335131/635d4b2c-0ea0-479a-a32b-85797b066367)

Dans le bloc de solution, il n'y avait déjà pas de `label` 🤷🏻‍♂️

Reply:
Je viens de rebase sur develop, ça a pris en compte les modifs que tu as apporté avec les composants Pix-UI, ça devrait régler le problème !

Reply:
C'est résolu, les ids peuvent rester 🙌

Reply:
Mais du coup maintenant, on aura le aria-label qui overide le label non ?